plugins/orchestrator-backend/src/service/SonataFlowService.ts

```diff
@@ -38,9 +38,8 @@ export class SonataFlowService {
         );
         return json;
       }
-      const responseStr = JSON.stringify(response);
-      this.logger.error(
-        `Response was NOT okay when fetch(${urlToFetch}). Received response: ${responseStr}`,
+      throw new Error(
+        this.getFetchErrorMessage(urlToFetch, 'fetchWorkflowInfoOnService'),
```

Comment:
I would have used `GET ` instead of `fetchWorkflowInfoOnService`

Reply:
I'm ok with that, but not sure if the context of this PR is the right one. I think it's more a refectory activity and it should be "extended" to other functions in this file. WDYT?

Reply:
I am in favor of putting the name of the operation and not the HTTP method. But...it is easy to put both: 'fetchWorkflowInfoOnService (GET)'. The more info the better